Library/.rubocop.yml

```diff
@@ -28,7 +28,7 @@ FormulaAuditStrict/Options:
   Enabled: true
 
 NewFormulaAudit/Options:
-  Enabled: false
+  Enabled: true 
```

Comment:
The editor plugin finds `.rubocop.yml` file at `HOMEBREW_LIBRARY` and uses it to highlight violations. 
So, in `HOMEBREW_LIBRARY/.rubocop.yml` ,`FormulaAuditStrict` and `NewFormulaAudit` departments can be disabled. This configuration can be inherited and overriden in another configuration file with the former departments(or specific cops) enabled, which will now be used in `style.rb` to handle `brew audit` and `brew style` commands. 

This also achieves the zero configuration goal for end user/developer. I can push it here if you want to see this approach.